src/components/auth/SignUp.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useEffect, useState } from "react";
```

Comment:
```suggestion
import { useEffect, useState } from "react";
```

also run `pnpm lint` and fix those linting errors